frontend/server/src/main/java/org/pytorch/serve/http/api/rest/ManagementRequestHandler.java

```diff
@@ -74,7 +75,7 @@ public void handleRequest(
                     if (HttpMethod.GET.equals(method)) {
                         handleListModels(ctx, decoder);
                         return;
-                    } else if (HttpMethod.POST.equals(method)) {
+                    } else if (HttpMethod.POST.equals(method) && isModeExplicit()) {
```

Comment:
We'll need to include this check for gRPC endpoints as well
https://github.com/pytorch/serve/blob/dd1f8dc36efe21f036be1e65043310dec74af28e/frontend/server/src/main/java/org/pytorch/serve/grpcimpl/ManagementImpl.java#L104

Reply:
Created issue for gRPC

Reply:
https://github.com/pytorch/serve/issues/3180